src/components/controls/MunicipalityDropdown.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Dropdown from './Dropdown';
+import municipalities from '../../dictionaries/municipalities';
+
+interface IMunicipalityDropdownProps {
+    value: string;
+    onChange: (val: string) => void;
+    disabled?: boolean;
+    label: string;
+}
+
+const municipalityDropdown = ({ value, onChange, disabled, label }: IMunicipalityDropdownProps) => (
+    <Dropdown
+        disabled={disabled}
```

Comment:
This is a bit excessively linewrapped :)

Reply:
Is it? :)

Reply:
What if I wrap code, to make it look like I have written more lines??

Reply:
fixed